src/components/pages/authentication/forms/LoginForm.js

```diff
@@ -119,6 +120,14 @@ class LoginForm extends Component {
 
 						analytics.identify({ ...newUser, method: "login" });
 
+						if (isReactNative() && useNewMessaging()) {
+							const loginData = {
+								refresh_token,
+								access_token
+							};
+							sendReactNativeMessage(loginData, MESSAGE_TYPES.STRIPE);
```

Comment:
Add a new messageType of AUTH in the reactNative helper.
```suggestion
							sendReactNativeMessage(loginData, MESSAGE_TYPES.AUTH);
```